src/shared/components/QueryBuilder/domain/helpers/queryToFilters.ts

```diff
@@ -48,23 +48,23 @@ export function queryToFilters(query: string): Filters {
         };
       }
 
-      const shouldChangeToInOperator = operator === OperatorKind['=~'] && value.includes('|');
-      if (shouldChangeToInOperator) {
-        return {
-          id: nanoid(10),
-          type: FilterKind['attribute-operator-value'],
-          active: true,
-          attribute: { value: attribute, label: attribute },
-          operator: { value: OperatorKind.in, label: OperatorKind.in },
-          value: {
-            value: value,
-            label: value
-              .split('|')
-              .map((v) => v.trim())
-              .join(', '),
-          },
-        };
-      }
+      // const shouldChangeToInOperator = operator === OperatorKind['=~'] && value.includes('|');
```

Comment:
Not strictly related to this PR, but if we let this code being executed, the "in" operator will be rendered in the filters in the UI. This cannot be because [it's not included in the operators dropdown options](https://github.com/grafana/explore-profiles/blob/main/src/shared/components/QueryBuilder/infrastructure/operatorsRepository.ts#L12).

Reply:
IMHO We should always remove the commented code or at least make it very clear with an extra comment why it is there and when this code comes back.

Reply:
Indeed, done!